lib/rubrowser/a.rb

```diff
@@ -0,0 +1,23 @@
+class A
+  def initialize
+    B.do
+  end
+
+  def self.do; end
+end
+
+class B
+  def initialize
+    C.do
+  end
+
+  def self.do; end
+end
+
+class C
+  def initialize
+    A.do
+  end
+
+  def self.do; end
+end
```

Comment:
May I ask where is this file being used? as I can't find a require for it anywhere.